Yoast/wordpress-seo
# Add a Academy Courses menu item to Yoast SEO settings

## Summary

This PR can be summarized in the following changelog entry:

* Adds a Courses Page to Yoast SEO, showing an overview of the available online courses offered by Yoast Academy. 

## Relevant technical choices:

* Moved the card components to Yoast-Components, see #[788](https://github.com/Yoast/yoast-components/pull/788)
* At the moment these cards are rendered by using sample data, but in the future they will get their input from an RSS feed. 

## Test instructions

This PR can be tested by following these steps:

* Checkout the following branch in Yoast-Components: `Add-Courses-Cards`, see [#788](https://github.com/Yoast/yoast-components/pull/788) for more info. 
* Run `yarn link yoast-components` in this branch. 
* Build this branch
* In the SEO menu in the admin panel, at the bottom you will see a new option "Courses"
* Click here and you will see an overview with all the courses Yoast Academy offers.
* Compare to the original issue if the design is correctly implemented and the links go to the correct places. Make sure to also check @moorscode additions to the spec request [here](https://github.com/Yoast/wordpress-seo/pull/11720#issuecomment-447240209)
* Check if the feed is correctly passed the licence by inspecting the page, opening the "Network" tab and searching for the feed request. In free "free" should be passed, in premium "premium". See screenshot below: 
<img width="965" alt="screenshot 2018-12-18 at 11 36 32" src="https://user-images.githubusercontent.com/16388834/50148761-9e53ad00-02b9-11e9-983a-7d20c1858a99.png">

## Quality assurance


Fixes #[1877](https://github.com/Yoast/wordpress-seo-premium/issues/1877)
